tests/make/make.spec.js

```diff
@@ -2,207 +2,157 @@ const { spawnSync } = require('child_process');
-        const uid = useUid ? '1000' : defaultValues.HOST_UID;
-        const email = useEmail ? 'email' : defaultValues.SUPERUSER_EMAIL;
-        const userName = useName ? 'name' : defaultValues.SUPERUSER_USERNAME;
-
-        const args = ['docker_compose_config'];
-
-        if (useVersion) args.push(`DOCKER_VERSION=${version}`);
-        if (useUid) args.push(`HOST_UID=${uid}`);
-        if (useEmail) args.push(`SUPERUSER_EMAIL=${email}`);
-        if (useName) args.push(`SUPERUSER_USERNAME=${userName}`);
-
-        runCommand(['create_env_file']);
-        const result = runCommand(args);
-        const {
-          services: { web },
-        } = JSON.parse(result.stdout);
-
-        expect(web.image).toStrictEqual(`mozilla/addons-server:${version}`);
-        expect(web.platform).toStrictEqual('linux/amd64');
-        expect(web.environment.HOST_UID).toStrictEqual(uid);
-        expect(web.environment.SUPERUSER_EMAIL).toStrictEqual(email);
-        expect(web.environment.SUPERUSER_USERNAME).toStrictEqual(userName);
-
-        const builder = 'test_builder';
-        const progress = 'test_progress';
-
-        let expectedBuildargs = `docker buildx bake web --progress=${progress} --builder=${builder}`;
-        const dockerBuildArgs = [
-          'docker_build_args',
-          `DOCKER_BUILDER=${builder}`,
-          `DOCKER_PROGRESS=${progress}`,
-        ];
-
-        if (usePush) {
-          expectedBuildargs += ' --push';
-          dockerBuildArgs.push('DOCKER_PUSH=true');
-        } else {
-          expectedBuildargs += ' --load';
-        }
-
-        const { stdout: actualBuildArgs } = runCommand(dockerBuildArgs);
-        expect(actualBuildArgs.trim()).toContain(expectedBuildargs.trim());
-
-        const { stdout: bakeConfigOutput } = runCommand([
-          'docker_build_config',
-        ]);
-        const bakeConfig = JSON.parse(bakeConfigOutput);
-        expect(bakeConfig.target.web.platforms).toStrictEqual(['linux/amd64']);
-      });
-    },
+
+  const version = require(path.join(rootPath, 'version.json'));
+
+  expect(version.version).toStrictEqual('version');
+  expect(version.commit).toStrictEqual('123');
+  expect(version.build).toStrictEqual('https://');
+  expect(version.source).toStrictEqual(
+    'https://github.com/mozilla/addons-server',
   );
+});
 
-  test('docker compose substitution', () => {
-    expect(new Set(Object.keys(defaultValues))).toStrictEqual(new Set(keys));
+test('map docker compose config', () => {
+  const values = {
+    DOCKER_VERSION: 'version',
+    HOST_UID: 'uid',
+    SUPERUSER_EMAIL: 'email',
+    SUPERUSER_USERNAME: 'name',
+  };
 
-    const composeFiles = globSync('docker-compose*.yml', { cwd: rootPath });
-    const variableDefinitions = [];
+  runSetup(values);
 
-    for (let file of composeFiles) {
-      const fileContent = fs.readFileSync(path.join(rootPath, file), {
-        encoding: 'utf-8',
-      });
+  const { stdout: rawConfig } = spawnSync(
```

Comment:
We can call docker explicitly here to test the scenario where a user doesn't use make and must rely only on .env or env variables.